packages/web-components/src/global/settings.scss

```diff
@@ -0,0 +1,2 @@
+@import 'uswds-helpers/src/styles';
+@import 'usa-form-group/src/styles';
```

Comment:
Do we need a global import like this? Would it be better for us to simply import single modules that are needed on a component to component basis (ie. less "stuff" will be loading that might not necessarily be needed by the component)?

Reply:
We can definitely start there and if we need to refactor to a global import we could in the future